_sass/landing/landing.scss

```diff
@@ -1,20 +1,25 @@
-.body {
-  padding: 0 $md;
-
-  @include media(sm) {
-    padding: 0 $sm;
-  }
+.home {
+  flex-wrap: wrap;
+  padding: $lg $md;
 
+  .h1 {
+    font-weight: $weight-bold;
+    margin-top: 0;
```

Comment:
![image](https://cloud.githubusercontent.com/assets/7440805/25551661/e2e3e1ec-2c3c-11e7-804c-cfe42c137ad9.png)
I know we talked about adding margin below this h1, but I actually think it looks better without it. It feels odd to me that the heading it closer to the hr than to the content.